cm/bin/cm_hmatrix.cc

```diff
@@ -0,0 +1,31 @@
+#include "CLog.hh"
+#include <TFile.h>
+#include <TString.h>
+#include "CMReconstruction.hh"
+#include "CmdLineConfig.hh"
```

Comment:
Group includes in groups (separated by empty line) in the following order:
standard libraries, root headers, other libraries, project headers.